src/components/_shared/Map/Map.js

```diff
@@ -6,20 +6,55 @@ import { BingLayer } from 'react-leaflet-bing-v2'
 
 import trackEvent from 'lib/analytics'
 import berlinMask from 'data/berlin-mask.geo.json'
+import brandenburgMask from 'data/brandenburg-mask.geo.json'
```

Comment:
The way this is implemented now loads both masks, even though only one is needed at a time. Given that the size of the added mask is negligible with ~7kb, I think that's fine though.